.releaserc.json

```diff
@@ -1,14 +1,20 @@
 {
   "branches": [
-    "main",
-    "master",
     {
-      "name": "beta",
+      "name": "master",
       "prerelease": true
     },
     {
-      "name": "rc",
-      "prerelease": true
+      "name": "release-v11",
+      "channel": "latest"
+    },
+    {
+      "name": "release-v10",
+      "range": "10.x"
+    },
+    {
+      "name": "release-v9",
+      "range": "9.x"
     }
```

Comment:
Let's align the `channel` name:
- https://github.com/GetStream/stream-chat-react/blob/release-v10/.releaserc.json
- https://github.com/GetStream/stream-chat-react/blob/release-v9/.releaserc.json

Reply:
Thanks for pointing this out

Reply:
I have also updated the channel for "master" to be "rc" to make it consistent with Anton's last pre-releases.

Reply:
nice! thanks for taking care of this!